test/jdk/java/nio/channels/Selector/SelectWithConsumer.java

```diff
@@ -308,11 +308,10 @@ public void testTimeout() throws Exception {
             Pipe.SinkChannel sink = p.sink();
             source.configureBlocking(false);
             source.register(sel, SelectionKey.OP_READ);
-            long start = System.currentTimeMillis();
+            long start = millisTime();
             int n = sel.select(k -> assertTrue(false), 1000L);
-            long duration = System.currentTimeMillis() - start;
+            expectDuration(start, 501, Long.MAX_VALUE);
```

Comment:
501 looks a bit strange here, I realise it's the same as > 500 that was there, just may be less surprising to see 500.